pkg/credentials/factory.go

```diff
@@ -72,17 +72,17 @@ func (s *StoreFactory) NewStore(credCtxs []string) (CredentialStore, error) {
 		return nil, err
 	}
 	if s.file {
-		return withOverride{
-			target: Store{
+		return &withOverride{
```

Comment:
Do I understand correctly that this change was made to avoid copying the mutex?

Reply:
Yep